src/bindings.rs

```diff
@@ -60,71 +60,111 @@ pub struct wiringPiNodeStruct {
 impl ::std::default::Default for wiringPiNodeStruct {
     fn default() -> Self { unsafe { ::std::mem::zeroed() } }
 }
+
+#[cfg(not(feature = "development"))]
 #[link(name = "wiringpi", kind = "static")]
 extern "C" {
     pub static mut piModelNames: [*const ::libc::c_char; 16usize];
     pub static mut piRevisionNames: [*const ::libc::c_char; 16usize];
     pub static mut piMakerNames: [*const ::libc::c_char; 16usize];
     pub static mut piMemorySize: [::libc::c_int; 8usize];
     pub static mut wiringPiNodes: *mut wiringPiNodeStruct;
-}
-#[link(name = "wiringpi", kind = "static")]
-extern "C" {
+
     pub fn wiringPiFailure(fatal: ::libc::c_int,
                            message: *const ::libc::c_char, ...)
      -> ::libc::c_int;
     pub fn wiringPiFindNode(pin: ::libc::c_int) -> *mut wiringPiNodeStruct;
     pub fn wiringPiNewNode(pinBase: ::libc::c_int, numPins: ::libc::c_int)
      -> *mut wiringPiNodeStruct;
-    pub fn wiringPiSetup() -> ::libc::c_int;
-    pub fn wiringPiSetupSys() -> ::libc::c_int;
-    pub fn wiringPiSetupGpio() -> ::libc::c_int;
-    pub fn wiringPiSetupPhys() -> ::libc::c_int;
-    pub fn pinModeAlt(pin: ::libc::c_int, mode: ::libc::c_int);
-    pub fn pinMode(pin: ::libc::c_int, mode: ::libc::c_int);
-    pub fn pullUpDnControl(pin: ::libc::c_int, pud: ::libc::c_int);
-    pub fn digitalRead(pin: ::libc::c_int) -> ::libc::c_int;
-    pub fn digitalWrite(pin: ::libc::c_int, value: ::libc::c_int);
-    pub fn pwmWrite(pin: ::libc::c_int, value: ::libc::c_int);
-    pub fn analogRead(pin: ::libc::c_int) -> ::libc::c_int;
-    pub fn analogWrite(pin: ::libc::c_int, value: ::libc::c_int);
-    pub fn wiringPiSetupPiFace() -> ::libc::c_int;
-    pub fn wiringPiSetupPiFaceForGpioProg() -> ::libc::c_int;
-    pub fn piBoardRev() -> ::libc::c_int;
+}
```

Comment:
I think the items in this block are unused. In that case you may just as well remove them, if you want to.